README.md

```diff
@@ -240,6 +259,49 @@ Use the following command to view the database within the MongoDB container.
 docker exec -it rosalution_rosalution-db_1 mongosh rosalution_db
 ```
 
+### Production
+
+#### Using the Build Script
+
+When deploying the Rosalution system in a production environment, it is important to ensure that the build is optimized
+ for security when deploying the Rosalution system in a production environment. This can be achieved by using the
+ `build.sh` script.
+
+The build.sh script is a command-line tool that can be used to build the Rosalution system for a production environment.
+ The script accepts various command-line arguments that can be used to customize the build process.
+
+The arguments that can be passed to the script include:
```

Comment:
It's nice to have a detailed layout of the build script arguments, usage should be left in the build script itself. 

As a result, these could be removed and the reader should be referred (even linked) to the build script itself for usage.

Reply:
I have made this requested change. its on line 275 of the readme. as well as linking to it with the first mention of it.